insecure/fixtures.go

```diff
@@ -59,12 +61,13 @@ func EgressMessageFixture(t *testing.T, codec network.Codec, protocol Protocol,
 	// creates corresponding event of that message that
 	// is sent by orchestrator network to orchestrator.
 	e := &EgressEvent{
-		CorruptOriginId:   originId,
-		Channel:           channel,
-		FlowProtocolEvent: content,
-		Protocol:          protocol,
-		TargetNum:         targetNum,
-		TargetIds:         targetIds,
+		CorruptOriginId:     originId,
+		Channel:             channel,
+		FlowProtocolEvent:   content,
+		FlowProtocolEventID: eventID.Hex(),
```

Comment:
Please keep `FlowProtocolEventID` as a `flow.Identifier`. Instead use the [`HexStringToIdentifier`](https://github.com/onflow/flow-go/blob/50a959a101a65ea37bf520b1adf60db934089949/model/flow/identifier.go#L39) here to convert the `hash` object into a `flow.Identifier`.

Reply:
Got it thanks

Reply:
https://github.com/onflow/flow-go/pull/3580/commits/3e1b9c9ac04b89ba2d1718d41d62c356c410b001